java-security-test/src/test/java/com/sap/cloud/security/test/extension/SecurityTestExtensionTest.java

```diff
@@ -63,7 +63,7 @@ public void addingStubIsPossibleAfterSetup(SecurityTestContext context) throws I
 						.willReturn(aResponse().withBody("OK")));
 
 		try (CloseableHttpResponse response = httpClient.execute(new HttpGet(url))) {
```

Comment:
Since Apache HTTP Client 5, one should use non-deprecated execute method by supplying a Response Handler. As discussed, we will overhaul these calls in the next commits.